cli/cmd/release_lint.go

```diff
@@ -13,18 +14,19 @@ func (r *runners) InitReleaseLint(parent *cobra.Command) {
 		Use:   "lint",
 		Short: "Lint a YAML",
 		Long:  "Lint a YAML",
+		SilenceUsage: true,
 	}
-	cmd.Hidden=true; // Not supported in KOTS 
 	parent.AddCommand(cmd)
 
 	cmd.Flags().StringVar(&r.args.lintReleaseYaml, "yaml", "", "The YAML config to lint. Use '-' to read from stdin.  Cannot be used with the `yaml-file` flag.")
 	cmd.Flags().StringVar(&r.args.lintReleaseYamlFile, "yaml-file", "", "The file name with YAML config to lint.  Cannot be used with the `yaml` flag.")
```

Comment:
For clarity, I'm wondering if it makes sense to remove the remove yaml and yaml-file usage options (but maybe still keep them for backwards compatibility)? For KOTS, a single YAML file/blob doesn't make sense.